Gruntfile.js

```diff
@@ -50,7 +50,7 @@ module.exports = function(grunt) {
     // https://www.npmjs.com/package/grunt-build-control
     buildcontrol: {
       options: {
-        dir: 'build', // Which directory to deploy
+        dir: 'build/content', // Which directory to deploy
```

Comment:
`build/content` doesn't exist since using move-up... Switching back to just `build`.